index.d.ts

```diff
@@ -331,6 +331,12 @@ export declare interface TracerOptions {
        */
       rules?: SamplingRule[]
     }
+
+    /**
+     * Whether to enable the experimental `getRumData` method.
+     * @default false
+     */
+    rumInjection?: boolean
```

Comment:
The feature does not inject. Maybe `enableGetRumData`? I guess in any case the flag will be removed in the future, but in the meantime it should not cause confusion.

Reply:
Doesn't look like this was actually renamed.